src/aind_smartspim_data_transformation/smartspim_job.py

```diff
@@ -180,11 +180,31 @@ def _write_stacks(self, stacks_to_process: List) -> None:
                 # this up
                 shutil.rmtree(ome_zarr_stack_path)
 
+    def _upload_derivatives_folder(self):
+        """
+        Uploads the derivatives folder inside of
+        the SPIM folder in the cloud.
+        """
+        s3_derivatives_dir = f"{self.job_settings.s3_location}/derivatives"
+        derivatives_path = Path(self.job_settings.input_source).joinpath(
+            "derivatives"
+        )
+        if self.job_settings.s3_location is not None:
+            logging.info(
+                f"Uploading {derivatives_path} to {s3_derivatives_dir}"
+            )
+            utils.sync_dir_to_s3(derivatives_path, s3_derivatives_dir)
+            logging.info(f"Removing: {derivatives_path}")
```

Comment:
I don't think we want to remove the derivatives path here. You can probably just log that the upload finished if you want.